src/content/1.7/development/internationalization/translation/native-module-translation.md

```diff
@@ -1,157 +1,19 @@
-</li>
-<!-- Block mymodule -->
-<div id="mymodule_block_left" class="block">
-  <h4>{l s='Welcome!' d='Modules.MyModule'}</h4>
-  <div class="block_content">
-    <p>Hello,
-       {if isset($my_module_name) && $my_module_name}
-           {$my_module_name}
-       {else}
-           World
-       {/if}
-       !
-    </p>
-    <ul>
-      <li><a href="{$my_module_link}" title="Click this link">Click me!</a></li>
-    </ul>
-  </div>
-</div>
-<!-- /Block mymodule -->
-```
-
-...becomes:
-
-```html
-<li>
-  <a href="{$base_dir}modules/mymodule/mymodule_page.php" title="{l s='Click this link' d='Modules.MyModule'}">{l s='Click me!' d='Modules.MyModule'}</a>
-</li>
-<!-- Block mymodule -->
-<div id="mymodule_block_left" class="block">
-  <h4>{l s='Welcome!' d='Modules.MyModule'}</h4>
-  <div class="block_content">
-    <p>
-      {if !isset($my_module_name) || !$my_module_name}
-        {capture name='my_module_tempvar'}{l s='World' d='Modules.MyModule'}{/capture}
-        {assign var='my_module_name' value=$smarty.capture.my_module_tempvar}
-      {/if}
-      {l s='Hello %1$s!' sprintf=$my_module_name d='Modules.MyModule'}
-    </p>
-    <ul>
-      <li><a href="{$my_module_link}"  title="{l s='Click this link' d='Modules.MyModule'}">{l s='Click me!' d='Modules.MyModule'}</a></li>
-    </ul>
-  </div>
- </div>
- <!-- /Block mymodule -->
-```
-
-...and the `display.tpl` file:
-
-```
-Welcome to this page!
-```
-
-...becomes:
-
-```
-{l s='Welcome to this page!' d='Modules.MyModule'}
-```
-
-Notice that we always use the `d` parameter. This is used by PrestaShop
-to assert which module the string belongs to. The translation tool needs
-it in order to match the string to translate with its translation. This
-parameter is mandatory for module translation.
-
-Translating complex code
-------------------------
-
-As we can see, the basis of template file translation is to enclose them
-in the `{l s='The string' mod='name_of_the_module'}`. The changes in
-`display.tpl` and in `mymodule.tpl`'s link and title texts are thus easy
-to understand. But added a trickier block of code for the "Hello World!"
-string: an if/else/then clause, and a text variable. Let's explore this
-code:
-
-Here is the original code:
-
-```
-Hello,
-  {if isset($my_module_name) && $my_module_name}
-    {$my_module_name}
-  {else}
-    World
-  {/if}
-!
-```
-As you can see, we need to get the "Hello World" string translatable,
-but also to cater for the fact that there is a variable. As explained in
-the "Translations in PrestaShop 1.5" chapter, variables are to be marked
-using `sprintf()` markers, such as `%s` or `%1$s`.
-
-Making "Hello %s!" translatable words in easy: we just need to use this
-code:
-
-```
-{l s='Hello %s!' sprintf=$my_module_name d='Modules.MyModule'}
-```
+In PrestaShop lingo, "native modules" refers to all the modules that are bundled and distributed with the Core. 
```

Comment:
`lingo` ?

Reply:
Yeah I searched it too. That means language or something like that

Reply:
https://dictionary.cambridge.org/fr/dictionnaire/anglais/lingo